.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+**What did you expect to happen:**
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+**Environment:**
+
+- Sonobuoy version: 
```

Comment:
is it worth it to add "(use `sonobuoy version`)" to the end of this?

Reply:
Yeah, I guess it would be good practice to get them to run a scan if they are having issues.